src/test/java/org/broadinstitute/hellbender/utils/pileup/ReadPileupUnitTest.java

```diff
@@ -78,22 +78,24 @@ public void testSplitByReadGroup() {
     @Test
     public void testGetPileupForSample() {
         // read groups and header
-        final SAMReadGroupRecord[] readGroups = new SAMReadGroupRecord[4];
+        final SAMReadGroupRecord[] readGroups = new SAMReadGroupRecord[5];
```

Comment:
add a note that readGroups[4] is left null intentionally - i like clarity in all things, including tests